cmd/crowdsec/api.go

```diff
@@ -15,6 +15,22 @@ func initAPIServer(cConfig *csconfig.Config) (*apiserver.APIServer, error) {
 		return nil, fmt.Errorf("unable to run local API: %s", err)
 	}
 
+	if hasPlugins(cConfig.API.Server.Profiles) {
+		log.Info("initing plugin broker")
+		err = pluginBroker.Init(cConfig.API.Server.Profiles, cConfig.ConfigPaths)
```

Comment:
When crowdsec is reloaded, old plugin processes are not killed, which leads to having 'dead' processes:
```
/tmp/crowdsec/crowdsec-v1.0.0-rc5/tests (plugins ✘)✭ ᐅ ps aux | grep notification-slack | wc -l
7
/tmp/crowdsec/crowdsec-v1.0.0-rc5/tests (plugins ✘)✭ ᐅ sudo kill -HUP 1587058
/tmp/crowdsec/crowdsec-v1.0.0-rc5/tests (plugins ✘)✭ ᐅ ps aux | grep notification-slack | wc -l
8

```

Reply:
Fixed via Fixed via https://github.com/crowdsecurity/crowdsec/pull/857/commits/5558c481975fdf496ad264253ab793a5764b027e